code/build-analysis/03b_county-census.R

```diff
@@ -0,0 +1,219 @@
+library(tidyverse)
+library(tidycensus)
+library(arrow)
+library(gt)
+
+calc_summary_stats <- function(data, variable) { #func to calculate sum stats
+  tibble(
+    Variable = variable,
+    Mean = mean(data[[variable]], na.rm = TRUE),
+    Median = median(data[[variable]], na.rm = TRUE),
+    SD = sd(data[[variable]], na.rm = TRUE)
+  )
+}
+
+calc_se <- function(x, y) {#calculate standard errors for difference in means
+  sqrt(var(x, na.rm = TRUE) / length(na.omit(x)) +
+         var(y, na.rm = TRUE) / length(na.omit(y)))
+}
+
+# set API
+# ideally, make this a secret for release
+census_api_key("5241b7834ca3181e92e126a6347110afd2c831d7",
```

Comment:
not that these really are proprietary, do we still want to be careful exposing a Census API key?

Reply:
I deleted it. It's probably going to expire in a year anyway